src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,20 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: $catalog-gap;
+  grid-template-columns: repeat(auto-fill, $card-width);
```

Comment:
You  had to add more media queries at this task. Firstly, you should have 4 cards in a row only when width i 1024px and more, but you have 4 cards when your width is 944px and more. You should have 3 cards in a row from 768px to 1024px, but you have 3 cards from 695px to 944px. You should have 2 cards in a row from 488px to 768px, but you have 2 cards from 448px to 695 px. And the last media query should be for width less than 488px, but your width for one card is 448px

Reply:
I used some media queries in the task Antisnake and here decided to try another tools. 
But ok - I added more media queries here as the second solution. )